plugins/woocommerce/tests/api-core-tests/ci-release.global-setup.js

```diff
@@ -61,10 +63,11 @@ test( `Setup remote test site`, async ( { page, request } ) => {
 		await page.goto( '/wp-admin/plugin-install.php' );
 		await page.getByRole( 'button', { name: Upload_Plugin } ).click();
 		await page.getByLabel( Plugin_zip_file ).setInputFiles( zipPath );
+		const uploadPromise = page.waitForResponse(
+			'/wp-admin/update.php?action=upload-plugin'
+		);
 		await page.getByRole( 'button', { name: Install_Now } ).click();
-		const uploadResponse = await page.waitForResponse(
-			'**/wp-admin/update.php?action=upload-plugin'
-		);;
+		const uploadResponse = await uploadPromise;
```

Comment:
It turns out that we **really** need to set up the network wait even before clicking the "Install Now" button.